opentracing/src/main/java/io/opentracing/Tracer.java

```diff
@@ -87,15 +87,18 @@
       /** Same as {@link Span#setTag(String, String)}, but for the span being built. */
       SpanBuilder withTag(String key, String value);
 
-      /** Same as {@link Span#setTag(String, String)}, but for the span being built. */
+      /** Same as {@link Span#setTag(String, boolean)}, but for the span being built. */
       SpanBuilder withTag(String key, boolean value);
 
-      /** Same as {@link Span#setTag(String, String)}, but for the span being built. */
+      /** Same as {@link Span#setTag(String, Number)}, but for the span being built. */
       SpanBuilder withTag(String key, Number value);
 
       /** Specify a timestamp of when the Span was started, represented in microseconds since epoch. */
       SpanBuilder withStartTimestamp(long microseconds);
 
+      /** Same as {@link Span#setBaggageItem(String, String)}, but for the span being built. */
+      SpanBuilder withBaggage(String key, String value);
```

Comment:
Yes that is correct.

I've done that [here](https://github.com/opentracing/opentracing-java/pull/20/files#diff-ec94bc6d9b32af57ad8009b28bfbb28eR45)